src/routes/_layout.svelte

```diff
@@ -13,6 +14,8 @@
   <a href="#main-content">Skip to Main Content</a>
 </div>
 
+<SiteAlert />
```

Comment:
@mukhtyar can we only append this when "beta" is in the subdomain? Or via a flag via `process.env.BETA`? Might be helpful to not see this while developing locally and on the dev site.

Reply:
@mukhtyar if you merge recent changes from `main` into this branch you should be able to  check for `process.env.showBetaBanner`

Reply:
Fixed via https://github.com/berkeley-gif/caladapt-website-2021/pull/64/commits/79841d5f0932e6a763e25560e9495a3820b8fe5a